translating-guide/en/customize-locale.md

```diff
@@ -0,0 +1,96 @@
+
+Please see [How Languages and Locales Work](https://docs.pkp.sfu.ca/translating-guide/en/managing-languages#how-languages-and-locales-work) in this guide for an explanation of how languages and locale files work in PKP software. 
+
+## Install and Enable the Custom Locale Plugin
+
+First, ensure that the plugin is installed and enable it:
+1. Go to Settings > Website > Plugins
+2. Find the Custom Locale Plugin in the list of Installed Plugins (if it is not listed with the Installed Plugins, check the Plugin Gallery and you may need to install it first)
+3. Click the box to enable it
+4. Click the blue arrow next to the plugin name and then the *Customize* link that appears below it. This will refresh the page and now you should see a new tab in the Website Settings menu called *Locales*.
+
+## How the Custom Locale Plugin Works
+
+Under the Locales tab you will see a list of locale files that contain all of the message keys that appear as text on your site, for the locale you have set as the Primary Locale under Website Settings > Setup > Languages. So, if you have more than one language on your site but Português (Brasil) is the Primary Locale, the locale files you see in the Locales tab will be for the Português (Brasil) language.  
+
+![](./assets/translating-guide-custom-locale-localefiles.png)
+
+If you want to edit the locale files for a different language installed on your site, you will need to temporarily set it as the Primary Locale and then change the Primary Locale when you’re finished editing the locales.
+
+![](./assets/translating-guide-primary-locale.png)
+
+You can open a file, find a message key in it, and enter a custom value to change how that piece of text appears on your site.  These files correspond to the locale files in the locale folders of your site installation.  The plugin provides a way to access these files directly.
+
+Each file in a locale folder is a collection of message keys categorized by topic, based on what feature or function they pertain to in the software application.  For example, most message keys related to an editor’s work are in the locale/en_US/editor.po file, while message keys for the ORCID profile plugin are in the plugins/generic/orcidProfile/locale/en_US/locale.po file. 
+
+An additional tricky thing to keep in mind is that every installation of OJS, OMP, or OPS has 2 sets of locale files – 1 in a folder called pkp-lib/locale/, which are for files that are common to all PKP software applications, and 1 in a folder called ojs/locale (or pkp/omp/locale or pkp/ops/locale), which are for files that are specific to that application.  For example, all of the English OJS email templates are in the ojs/locale/en_us/emails.po file. Whereas all of the English message keys related to user accounts are in the pkp-lib/locale/en_us/user.po file.
```

Comment:
- Change `pkp-lib/locale/` to `lib/pkp/locale`
- Change `ojs/locale` to `locale` (and remove references to `ops/locale` and `omp/locale`)

You're referring to the git repository names, which won't be relevant for users.

Reply:
Thanks, I found this confusing.